spec.html

```diff
@@ -34530,13 +34530,13 @@ <h1>Static Semantics: Early Errors</h1>
             It is a Syntax Error if the List of Unicode code points that is SourceText of |UnicodePropertyName| is not identical to a List of Unicode code points that is a Unicode property name or property alias listed in the &ldquo;Property name and aliases&rdquo; column of <emu-xref href="#table-nonbinary-unicode-properties"></emu-xref>.
```

Comment:
Note: You *could* simplify the tables of properties: List only one name for each property, no further aliases, and just point to PropertyAliases.txt for aliases.